Makefile

```diff
@@ -24,8 +24,13 @@ fmt:
 	gofmt -d -s .
 
 .PHONY: check
-check: all test lint
+check: all test lint fuzz
```

Comment:
This is a frequently run target. Won't it be a bit too long with fuzzing?
I suggest we keep fuzzing as a separate target.

Reply:
In the absence of a dedicated script for making a release, this looked like the second best place. But if you're saying that this is a target that's used in your development cycle, I agree it's too much.

Anyway, removed.